ykrt/src/location.rs

```diff
@@ -204,6 +204,21 @@ pub(crate) struct HotLocation {
     pub(crate) trace_failure: TraceFailureThreshold,
 }
 
+impl HotLocation {
+    /// Mark a trace at this `HotLocation` as having failed. The return value indicates whether
```

Comment:
nitpick: "trace *starting* at this location"

Reply:
Fixed in bf7ff37.